server.test.js

```diff
@@ -30,17 +35,99 @@ describe('Games', () => {
     // write a beforeEach hook that will populate your test DB with data
     // each time this hook runs, you should save a document to your db
     // by saving the document you'll be able to use it in each of your `it` blocks
+    const newGame = new Game({
+      title: 'fifa 2018',
+      genre: 'soccer',
+      releaseDate: 'JAN 2018'
+
+    });
+    newGame.save((err, savedGame) =>{
+      if(err){
+        console.log(err);
+        done();
+      }
+      GameId = savedGame._id.toString();
+      done();
+    })
+
   });
   afterEach(done => {
     // simply remove the collections from your DB.
+    Game.remove({}, err => {
+      if (err) console.log(err);
+      done();
+    });
   });
 
   // test the POST here
+  describe(` [POST]  /api/game/create`, () => {
+    it('should save the game to the db', done =>{
+      chai.request(server)
+      .post('/api/game')
+      .send({ name: 'fifa2019', genre: 'soccer', releaseDate: 'Jan 2019'})
+      .then(response => {
+
+done();
+      })
+      .catch(err =>{
+        throw err;
+      });
+      it('should fail if no name or genre is provided', (done) =>{
+        chai
+        .request(server)
+        .post('/api/game/create')
+        .send({ name: 'fifa2019',  releaseDate: 'Jan 2019'})
+        .then(response => {
+          expect(reponse.status).to.equal(200);
+          expect(response.body).to.include(name,genre,releaseDate);
+          done();
+        })
+        .catch(err =>{
+          throw err;
+          done();
+        });
+      
```

Comment:
try to limit empty space with closing curly braces or brackets.  There's nothing to separate in the code, so the closing braces should be on line 89 instead of 90.